src/base/bittorrent/peerinfo.cpp

```diff
@@ -408,3 +408,8 @@ QString PeerInfo::flagsDescription() const
 {
     return m_flagsDescription;
 }
+
+int PeerInfo::downloadingPieceIndex() const
```

Comment:
What is the main difference between the `TorrentHandle` and `TorrentInfo` classes from your point of view? Could you tell me, please?